examples/tutorial/step11.go

```diff
@@ -16,19 +17,48 @@ const flowN = 3
 
 func main() {
 	config := flow.Config{}
-	flow.SystemInit(&config)
+	err := flow.SystemInit(&config)
+	if err != nil {
+		log.Fatal(err)
+	}
+
 	initCommonState()
-	l3Rules = packet.GetL3ACLFromORIG("rules2.conf")
+
+	l3Rules, err = packet.GetL3ACLFromORIG("rules2.conf")
+	if err != nil {
+		log.Fatal(err)
+	}
 	go updateSeparateRules()
-	firstFlow := flow.SetReceiver(0)
-	outputFlows := flow.SetSplitter(firstFlow, mySplitter, flowN, nil)
-	flow.SetStopper(outputFlows[0])
-	flow.SetHandler(outputFlows[1], myHandler, nil)
+	firstFlow, err := flow.SetReceiver(0)
+	if err != nil {
+		log.Fatal(err)
+	}
+	outputFlows, err := flow.SetSplitter(firstFlow, mySplitter, flowN, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+	err = flow.SetStopper(outputFlows[0])
```

Comment:
It depends on the conventions, but I think that for error-only returns one can slightly simplify the code
and improve `err` variable locality by using init clause of `if`:
```go
// Before:
err = flow.SetHandler(outputFlows[1], myHandler, nil)
if err != nil {
	log.Fatal(err)
}
// After:
if err := flow.SetHandler(outputFlows[1], myHandler, nil); err != nil {
	log.Fatal(err)
}
```
It does not reduce real amount of code, but in my opinion it makes it easier to "scan through the code" with less clutter.